static/js/stream_settings_ui.js

```diff
@@ -3,10 +3,10 @@ import _ from "lodash";
 import tippy from "tippy.js";
 
 import render_unsubscribe_private_stream_modal from "../templates/confirm_dialog/confirm_unsubscribe_private_stream.hbs";
+import render_settings_stream_list from "../templates/stream_settings/settings_stream_list.hbs";
+import render_settings_stream_list_item from "../templates/stream_settings/settings_stream_list_item.hbs";
```

Comment:
Do we want to call these `browse_streams_list` and `browse_streams_list_item`?  I think "browse streams" is a reasonable name within the "stream settings" namespace for the left side of "stream settings" just like have "stream_edit" / "stream_create" for the right side.